src/table/__integ__/resizable-columns.test.ts

```diff
@@ -373,3 +379,42 @@ test.each([false, true])(
     );
   }
 );
+
+describe('percentage column widths', () => {
+  test(
+    'supports percentage column width',
+    setupTest(
+      async page => {
+        await expect(page.getColumnWidth(1)).resolves.toBe(234);
+        await expect(page.getColumnWidth(3)).resolves.toBe(350);
+      },
+      { percentageWidths: true },
+      { width: 1200 }
+    )
+  );
+
+  test(
+    'percentage column widths do not change on browser resize',
+    setupTest(
+      async page => {
+        await page.setWindowSize({ ...defaultScreen, width: 1400 });
```

Comment:
Why is width set here and also on line 405, as an argument?

Reply:
the second one is the "on browser resize" part (i.e. the page has already been rendered in `setupTest`, this is now changing the width with a fully-rendered table